docs/specs/tripgo.swagger.yaml

```diff
@@ -2015,6 +2000,63 @@ definitions:
       - alt
       - localIcon
 
+  ModeIdentifier:
+    description: This is a map/dictionary with multiple items of this type. Each key is a mode identifier. See https://developer.tripgo.com/faq/#mode-identifiers.
+    properties:
+      title:
+        type: string
+      URL:
+        type: string
+      color:
+        $ref: '#/definitions/Color'
+      icon:
+        type: string
+        description: File name part for downloading an image for this mode. [See separate documentation](TODO) for how to construct a URL from this.
```

Comment:
There’s a TODO here. Also the name doesn’t match what’s below with localIcon vs remoteIcon.

Reply:
this is extracted from what we already had, that's not new

Reply:
should we change this to `localIcon` and deprecated `icon`?